Server/tools/cad/sketching.py

```
@@ -0,0 +1,156 @@
+        endpoint = get_endpoints("cad")["arc"]
+        headers = get_headers()
+        data = {
+            "point1": point1,
+            "point2": point2,
+            "point3": point3,
+            "plane": plane
+        }
+        return send_request(endpoint, data, headers)
+
+    except requests.RequestException as e:
+        logging.error("Draw arc failed: %s", e)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)